src/components/Login/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import styles from './loginForm.module.scss';
+import LoginCaption from './LoginCaption/LoginCaption';
+import LoginButton from './LoginButton/LoginButton';
+import TextField from '../../shared/TextField/TextField';
+import { getPassword, getUsername } from '../selectors';
+import { changePasswordValue, changeUsernameValue, formSubmit } from '../store/actions';
+import AlertMessage from '../../shared/AlertMessage/AlertMessage';
+
+const ERROR_MESSAGE = 'Wrong username or password. Please, try again';
+
+const LoginForm = ({
+  username, password, changePassword, changeUsername, submitForm
+}) => (
+  <div className={styles.loginForm}>
```

Comment:
I think it will be better to wrap it in form tag and add onSubmit handler so we can submit form either by click and by hitting Enter button